mkdocs/docs/HPC/migration_to_rhel9.md

```diff
@@ -0,0 +1,104 @@
+
+## Impact on central software stack
+
+The migration to RHEL8 as operating system should not impact your workflow, everything will basically be working as it did before (incl. job submission, etc.).
+
+However, there will be impact on the availability of software that is made available via modules.
+
+**Software that was installed with an older compiler toolchain will no longer be available once the clusters have been updated to RHEL9.**
+
+This includes all software installations on top of a toolchain that is older than foss/2023a, gompi/2023a, intel/2023a, iimpi/2023a, GCC(core)/12.3.0.
+
+The `module` command will produce a clear warning when you are loading modules that are using a toolchain that will no longer be available after the cluster has been migrated to RHEL8.
+for example:
+
+    foss/2019a:
+       ___________________________________
+      /  This module will soon no longer  \
+      \  be available on this cluster!    /
+       -----------------------------------
+             \   ^__^
+              \  (xx)\_______
+                 (__)\       )\/\
+                  U  ||----w |
+                     ||     ||
+    Only modules installed with a recent toolchain will still be available
+    when this cluster has been migrated to the RHEL 9 operating system.
+    Recent toolchains include GCC(core)/12.3.0, gompi/2023a, foss/2023a,
+    iimpi/2023a, intel/2023a, and newer versions.
+
+    You should update your workflow or job script to use more recent software
+    installations, or accept that the modules you currently rely on will soon
+    no longer be available.
+
+    To request a more recent version of the software you are using,
+    please submit a software installation request via:
+
+    https://www.ugent.be/hpc/en/support/software-installation-request
+
+    The HPC-UGent Tier-2 clusters running RHEL 8 will be migrated to RHEL 9.
+
+    For more information, see https://docs.hpc.ugent.be/migration_to_rhel9/
+
+    If you have any questions, please contact hpc@ugent.be .
+
+If you require software that is currently only available with an older toolchain on the HPC-UGent Tier-2 clusters that are still running RHEL8, check via "module avail" if a more recent version is installed that you can switch to, or submit a [software installation request](https://www.ugent.be/hpc/en/support/software-installation-request) so we can provide a more recent installation of that software which you can adopt.
+
+## Planning
+
+We plan to migrate the HPC-UGent Tier-2 clusters that are still using RHEL 8 to RHEL 9 one by one, following the schedule outlined below.
+
+| ***cluster*** | ***migration start*** | ***migration completed on*** |
+| --- | ---- | --- |
+| skitty | September 2024 | |
+| joltik | October 2024 | |
+| accelgor | November 2024 | |
+| gallade | December 2024 | |
+| donphan | january 2025 | |
```

Comment:
it will be February, and mention after 1st sem exams. also Make a not that donphan and doduo will be done at almost the same time